src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -62,14 +68,14 @@ public void setUp() {
     @Test
     public void givenCase_whenEventIsAmendApplication_thenCanHandle() {
         assertThat(handler
-                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CaseType.CONSENTED, EventType.AMEND_APP_DETAILS),
+                .canHandle(CallbackType.ABOUT_TO_SUBMIT, CONSENTED, EventType.AMEND_APP_DETAILS),
```

Comment:
Could make use the newly added helper method here - `uk.gov.hmcts.reform.finrem.caseorchestration.test.assertCanHandle`

Reply:
Utilised, thank you for the suggestion